docs/jwst/outlier_detection/outlier_detection_ifu.rst

```diff
@@ -0,0 +1,32 @@
+.. _outlier_detection_ifu_:
+
+
+Python Interface to OutlierDetectionIFU: OutlierDetectionIFU()
+=========================================================
+
+This module serves as the interface for applying outlier_detection to IFU
+ observations, like those taken with NIRISS and MIRI.  The code implements the
+ basic outlier detection algorithm used with HST data, as adapted to JWST IFU
+ observations.
+
+ Specifically, this routine performs the following operations:
+
+ * Extract parameter settings from input model and merge them with any user-provided values
+
+ * Resamples all input images into :py:class:`~jwst.datamodels.IFUCubeModel` observations.
+
+   - Resampled images will be written out to disk if ``save_intermediate_results`` parameter has been set to `True`
```

Comment:
The word "image" has so many connotations that immediately imply 2-D, but in this case we're talking about 3-D "images". Perhaps just say "cubes" instead of "images"?